KubeArmor/core/kubeArmor.go

```diff
@@ -611,6 +612,11 @@ func KubeArmor() {
 
 	policyService := &policy.ServiceServer{}
 
+	postureService := &posture.ServiceServer{}
+	postureService.UpdateDefaultPosture = dm.UpdateDefaultPosture
```

Comment:
dm.UpdateDefaultPosture is for namespace default posture and not the Global Default Posture. 

Namespace Default Posture depends on namespace annotations. But Global Default Posture is the default value for unannotated namespaces.

Currently we use cfg.GlobalCfg.DefaultPosture* as default values as can be seen at
https://github.com/kubearmor/KubeArmor/blob/7ff5ebdb23176cda21762f8e409f3f141090155e/KubeArmor/core/kubeUpdate.go#L387-L390

So ideally, the process to update global default posture would be
- Updating the global cfg,
- Loop through the default posture map and updating the respective defaults of unannotated pods (this might need an improvement of logic since updating individuals defaults might not be feasible)

Reply:
Thanks @daemon1024 would make changes